modules/tasks/tasks/src/main/resources/webapp/js/controllers.js

```diff
@@ -260,6 +260,11 @@
     controllers.controller('TasksManageCtrl', function ($scope, ManageTaskUtils, Channels, DataSources, Tasks, Triggers,
                 $q, $timeout, $stateParams, $http, $filter, ModalFactory, LoadingModal, HelpStringManipulation) {
 
+        $scope.showBubbles = true;
+        $scope.bubbleFunction = function() {
```

Comment:
Name of this function is not intuitive. Please change it to showHideBubbles for example.